addons/im_livechat/static/src/legacy/models/public_livechat_message.js

```diff
@@ -46,9 +46,6 @@ const PublicLivechatMessage = Class.extend({
         this._type = data.message_type || undefined;
 
         this._processAttachmentURL();
-        this._attachmentIDs.forEach(function (attachment) {
-            attachment.filename = attachment.filename || attachment.name || _t("unnamed");
-        });
```

Comment:
not sure if it's a safe removal. There is no attachment because of your previous hack, but we will want to re-enable the feature at some point. In that regard, you may want to keep track of the previous logic in order to reimplement the behaviours accurately when the time comes

Reply:
If we want to reintroduce the feature back, it will be the new discuss code way. There's already new code for handling attachments, so it will use it.
No deletion of code in this PR is livechat-specific at all, so there's no reason to keep the dead code at all.